samples/19.a.single-sign-on-for-enterprise-apps/bot/src/createBotAdapter.js

```diff
@@ -0,0 +1,19 @@
+const { BotFrameworkAdapter } = require('botbuilder');
+
+const {
+  MICROSOFT_APP_ID,
+  MICROSOFT_APP_PASSWORD
+} = process.env;
+
+module.exports = function createBotAdapter() {
+  const adapter = new BotFrameworkAdapter({
+    appId: MICROSOFT_APP_ID,
+    appPassword: MICROSOFT_APP_PASSWORD
+  });
+
+  // adapter.onTurnError = async (_, error) => {
```

Comment:
Should we uncomment this code?

Reply:
Maybe remove this code? I never see an error pumping thru this function (after I uncomment it). And I copy it from our Echo Bot sample.

It's not very visible even I uncomment it, that's why I am thinking to remove it.

Reply:
Removed.